src/client.rs

```diff
@@ -81,54 +132,23 @@ impl Client {
             hash_function: default_hash_function,
         })
     }
+}
+
+impl Client {
+    #[deprecated(since = "0.10.0", note = "please use `connect` instead")]
+    pub fn new<C: Connectable>(target: C) -> Result<Self, MemcacheError> {
+        return Self::connect(target);
+    }
 
     pub fn connect<C: Connectable>(target: C) -> Result<Self, MemcacheError> {
-        Self::with_pool_size(target, 1)
+        ClientBuilder::new(target).with_max_size(1).build()
     }
 
     fn get_connection(&self, key: &str) -> Pool<ConnectionManager> {
         let connections_count = self.connections.len();
         return self.connections[(self.hash_function)(key) as usize % connections_count].clone();
     }
 
-    /// Set the socket read timeout for TCP connections.
```

Comment:
Seems these two API was removed, can we keep it and add a deprecated annotation?

Reply:
We can't keep those APIs. Because if a `Connection` gets dropped, a new `Connection` will be created for the next request. But it won't have the read and write timeouts. The new APIs in `ClientBuilder` override the on_acquire hook to set the timeouts.